src/app/custom-navigation-mode/custom-navigation-mode.component.html

```diff
@@ -0,0 +1,82 @@
+<p>All of the wizards below use a custom navigation mode that allows the user to return to any steps he/she already visited in any direction.  This page illustrates several options to configure a wizard with a custom navigation mode.</p>
+
+<p>Option 1: Specify a custom <code>navigationModeFactory</code> in <code>ArchwizardModule.forRoot()</code> and use a custom navigation mode name.</p>
```

Comment:
I think we should add here that the `ArchwizardModule.forRoot()` call should be added to the module file, just to be sure it is understood correctly.

Reply:
Done in 22f2f6e.